.pipelines/e2e-tests-template.yml

```diff
@@ -28,55 +28,31 @@ jobs:
           - template: build-images.yml
 
           - script: |
-              echo "Installing aks-engine..."
-              curl -LO https://github.com/Azure/aks-engine/releases/download/$(AKS_ENGINE_VERSION)/aks-engine-$(AKS_ENGINE_VERSION)-linux-amd64.tar.gz
-              tar -zxvf aks-engine-$(AKS_ENGINE_VERSION)-linux-amd64.tar.gz
-              sudo mv aks-engine-$(AKS_ENGINE_VERSION)-linux-amd64/aks-engine /usr/local/bin/
-              echo
-
               echo "Installing kubectl..."
```

Comment:
This is always getting the latest kubectl version. As a follow up we should get the correct `kubectl` version based on the cluster version.